cmd/watchdog/main.go

```diff
@@ -87,35 +94,31 @@ func main() {
 
 	logger := slog.New(slog.NewTextHandler(os.Stdout, nil))
 
-	report.SetupSentry()
-	defer report.FlushSentry()
-
 
-	reporter := report.NewReporter(cfg.Env, version)
-	reporter.ConfigureScope()
 
 	cacheDir := "cache"
-	if err = createCacheDirectory(cacheDir, logger); err != nil {
+	if err = createCacheDirectory(cacheDir, logger , reporter); err != nil {
 		logger.Error("Failed to create cache directory", "error", err)
 		os.Exit(1)
 	}
 
 	// Download GTFS bundles for all servers on startup
-	downloadGTFSBundles(servers, cacheDir, logger)
+	downloadGTFSBundles(servers, cacheDir, logger , reporter)
 
 	app := &application{
 		config: cfg,
 		logger: logger,
+		reporter: reporter,
 	}
 
 	app.startMetricsCollection()
 
 	// Cron job to download GTFS bundles for all servers every 24 hours
-	go refreshGTFSBundles(servers, cacheDir, logger , 24 * time.Hour)
+	go refreshGTFSBundles(servers, cacheDir, logger , 24 * time.Hour , reporter)
```

Comment:
there's an extra space in between logger and the comma. it should look like this: `logger, reporter`